include/z64pause_menu.h

```diff
@@ -0,0 +1,248 @@
+    /* 0x220 */ f32 roll;
+    /* 0x224 */ u16 alpha;
+    /* 0x226 */ s16 offsetY;
+    /* 0x228 */ UNK_TYPE1 unk_228[0x8];
+    /* 0x230 */ s32 stickAdjX;
+    /* 0x234 */ s32 stickAdjY;
+    /* 0x238 */ s16 cursorPoint[PAUSE_PAGE_MAX]; // indexed by PauseMenuPage enum
+    /* 0x242 */ s16 cursorXIndex[PAUSE_PAGE_MAX]; // indexed by PauseMenuPage enum
+    /* 0x24C */ s16 cursorYIndex[PAUSE_PAGE_MAX]; // indexed by PauseMenuPage enum
+    /* 0x256 */ s16 unk_256; // Uses DungeonItem enum
+    /* 0x258 */ s16 cursorSpecialPos;
+    /* 0x25A */ s16 pageSwitchInputTimer; // Used to introduce a delay before switching page when arriving on the "scroll left/right" positions while holding stick left/right.
+    /* 0x25C */ u16 namedItem;
+    /* 0x25E */ u16 cursorItem[PAUSE_PAGE_MAX]; // indexed by PauseMenuPage enum
+    /* 0x268 */ u16 cursorSlot[PAUSE_PAGE_MAX]; // indexed by PauseMenuPage enum
+    /* 0x272 */ u16 equipTargetItem;
+    /* 0x274 */ u16 equipTargetSlot;
+    /* 0x276 */ u16 equipTargetCBtn; // PauseEquipCButton enum
+    /* 0x278 */ s16 equipAnimX;
+    /* 0x27A */ s16 equipAnimY;
+    /* 0x27C */ s16 equipAnimAlpha;
+    /* 0x27E */ s16 infoPanelOffsetY;
+    /* 0x280 */ u16 nameDisplayTimer;
+    /* 0x282 */ u16 nameColorSet;
+    /* 0x284 */ s16 cursorColorSet;
+    /* 0x286 */ s16 cursorSpinPhase;
+    /* 0x288 */ f32 cursorX;
+    /* 0x28C */ f32 cursorY;
+    /* 0x290 */ f32 cursorWidth;
+    /* 0x294 */ f32 cursorHeight;
+    /* 0x298 */ f32 cursorShrinkRate;
+    /* 0x29C */ s16 promptChoice; // save/continue choice: 0 = yes; 4 = no
+    /* 0x29E */ s16 promptAlpha;
+    /* 0x2A0 */ s16 ocarinaSongIndex;
+    /* 0x2A2 */ u8 worldMapPoints[20];
+    /* 0x2B6 */ u8 unk_2B6; // unused red?
+    /* 0x2B7 */ u8 unk_2B7; // unused green?
+    /* 0x2B8 */ u8 unk_2B8; // unused blue?
+    /* 0x2B9 */ u8 itemDescriptionOn; // helpful description of item given through a message box
+    /* 0x2BA */ s16 equipAnimScale; // scale of item icon while moving being equipped to c-button
+    /* 0x2BC */ s16 equipAnimShrinkRate; // rate the scale is shrinking for the item icon while moving being equipped to c-button
+    /* 0x2BE */ s16 ocarinaButtonsY[5];
+    /* 0x2C8 */ u16 unk_2C8; // Uses PauseMenuPage enum for Owl Warp. Never set.
+    /* 0x2CA */ s16 unk_2CA; // Uses OwlWarpId enum for Owl Warp. Never set.
+} PauseContext; // size = 0x2D0
+
+#define IS_PAUSE_STATE_GAMEOVER(pauseCtx) \
+    (((pauseCtx)->state >= PAUSE_STATE_GAMEOVER_0) && ((pauseCtx)->state <= PAUSE_STATE_GAMEOVER_10))
+
+#define IS_PAUSE_STATE_OWLWARP(pauseCtx) \
+    (((pauseCtx)->state >= PAUSE_STATE_OWLWARP_2) && ((pauseCtx)->state <= PAUSE_STATE_OWLWARP_6))
+
+#define IS_PAUSE_MAIN_STATE_SONG_PROMPT(pauseCtx) \
+    (((pauseCtx)->mainState >= PAUSE_MAIN_STATE_SONG_PROMPT_INIT) && \
+     ((pauseCtx)->mainState <= PAUSE_MAIN_STATE_SONG_PROMPT_DONE))
+
+#define IS_PAUSED(pauseCtx) \
```

Comment:
There are 4 more uses of these in `z_play.c`, coul you use these there?

Reply:
The code in `z_play.c` was not using `PAUSE_STATE_OFF`, that's why I missed it.
Thanks for checking!